### ISSUE TITLE
Windows limitations

### ISSUE THREAD

**Issue by n8crwlr** [bruhov/WinThumbsPreloader#12]
Thanks for this tool! 💯 

Can you explain where Windows saves the thumbs? I have around 10K images and it is preloading again and again, even if it feels there are almost all thumbs preloaded... Is the space **limited** in Windows?

If i run WTP again, it is thumbing again 30% of images for example.

Sometimes, i am not sure, if it is fully preloaded or not when opening a directory (even it looks fast).

**Comment by ACiDGRiM** [comment]
Windows stores thumbnails on network resources in a thumbs.db file, hidden in the folder of the images.
Local images are cached here: %userprofile%\AppData\Local\Microsoft\Windows\Explorer

I would like to add to this issue in that this application is processing thumbnails for folders with existing thumbnails. It would be beneficial to only process images without an existing thumbnail

**Comment by n8crwlr** [comment]
> I would like to add to this issue in that this application is processing thumbnails for folders with existing thumbnails. It would be beneficial to only process images without an existing thumbnail

Yes, it does, but not fully - on my files i got this for around 30 %, but only if there were new images. Again, here not only for new images. It`s just a bit curios what happens here
